src/compiler/jsgen.js

```diff
@@ -439,9 +439,29 @@ class JSGenerator {
             return new TypedInput(`p${node.index}`, TYPE_UNKNOWN);
 
         case 'compat':
+            if (node.blockType === BlockType.INLINE) {
```

Comment:
Currently `util.startBranch` with the `loop` parameter as true does not actually make it a loop, I dont know how to fix this while still getting the return value.

Reply:
And honestly I don't see a practical reason for the block to be a loop anyways but I did attempt to support it with no avail, the return value was able to be grabbed but without using `globalState.blockUtility._startedBranch[0]` I cant get the branch from what I know.